src/App.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import "./App.css";
+import Cards from "./components/Cards";
+import GameBoard from "./components/GameBoard";
+import ScoreBoard from "./components/ScoreBoard";
+
+const App = () => {
+  const [cards, setCards] = useState(Cards);
+  const [score, setScore] = useState(0);
+  const [highestScore, setHighestScore] = useState(0);
+
+  const handleCardClick = (e) => {
+    const clickedCard = cards.find((item) => item.logoURL === e.target.id);
+
+    if (!clickedCard.clicked) {
+      clickedCard.clicked = true;
+
+      setScore(score + 1);
```

Comment:
When updating state based on previous state, it's best to use functional updates to avoid unexpected behavior.

setScore(prevScore => prevScore + 1)